DigitalLearningSolutions.Web/Models/DelegateRegistrationData.cs

```diff
@@ -1,20 +1,29 @@
 ﻿namespace DigitalLearningSolutions.Web.Models
 {
     using System;
-    using DigitalLearningSolutions.Web.ViewModels.Register;
 
     public class DelegateRegistrationData
     {
         public DelegateRegistrationData()
         {
             Id = new Guid();
-            RegisterViewModel = new RegisterViewModel();
-            LearnerInformationViewModel = new LearnerInformationViewModel();
         }
 
         public Guid Id { get; set; }
-        public RegisterViewModel RegisterViewModel { get; set; }
-        public LearnerInformationViewModel LearnerInformationViewModel { get; set; }
         public string? PasswordHash { get; set; }

```

Comment:
I really like how you've grouped the fields per form section - do you think it's worth moving the passwordHash to the end to match the page order?